File: C#/L5/Title.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace L5
{

    public class Title : InOut
    {
        private string name;

        public Title()
        {
            name = "IT teacher";
        }

        public Title(string name)
        {
            this.name = name;
        }

        public string getName()
        {
            return name;
        }

        public void setName(string name)
        {
            this.name = name;
        }

        public void fill()
        {
            Console.WriteLine("\nTitle\nname(string):\n");
            name = Console.ReadLine();
        }

        public void toString()
        {
            Console.Write("\nTitle:\n" +
                    "Title = " + name + "\n");
        }
    }
}

File: JAVA/L3/P1/Title.java
package LABS.L3.P1;
//Должность
import java.util.Scanner;

public class Title {
    private String name;

    public Title(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void fill(Scanner scanner){
        name = scanner.nextLine();
    }

    @Override
    public String toString() {
        return "Title:" +
                "Name= " + name;
    }
}
